OpenSSL/SSL.py

```diff
@@ -65,6 +65,7 @@ class _buffer(object):
     pass
 
 OP_SINGLE_DH_USE = _lib.SSL_OP_SINGLE_DH_USE
+OP_SINGLE_ECDH_USE = SSL_OP_SINGLE_ECDH_USE
```

Comment:
oops, sorry. fixed it.

`SSL_OP_SINGLE_ECDH_USE` was introduced at same time as SSL_CTX_set_tmp_ecdh: openssl/openssl@ea262260469e49149cb10b25a87dfd6ad3fbb4ba
So, I think conditionally catching exception is unnecessary